.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.12.x
+  - 1.15.x
```

Comment:
Do we want to up the version here.  We were testing 1.12.x since we support Golang versions that old.

Reply:
`errors.As` and `errors.Is` are missing in 1.12.x, and the latest backoff library is using them. It's been almost two years since 1.12 was released, so at least we can put 1.13